app/Http/Resources/RequestNcpoResource.php

```
@@ -0,0 +1,28 @@
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'id' => $this->id,
+            'po_id' => $this->po_id,
+            'date' => $this->date,
+            'ncpo_no' => $this->ncpo_no,
+            'justification' => $this->justification,
+            'created_at' => $this->createdAtDateHuman,
+            'new_po_total' => $this->new_po_total,
+            'items' => RequestNcpoItemResource::collection($this->items),
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\RequestNcpoItemResource' in method 'toArray'. (undefined)

(StaticAccess)